vtex-apps/store-link
# Create link.product block

#### What does this PR do? \*

#### How to test it? \*
[workspace](https://quickview--storecomponents.myvtex.com/) click in a quickview inside the product-summary

You should read the docs too and see if it has a good description

#### Describe alternatives you've considered, if any. \*



#### Related to / Depends on https://github.com/vtex-apps/modal-layout/pull/11
